app/helpers/dashboards_helper.rb

```diff
@@ -1,25 +1,25 @@
 # Convenience methods consumed in the dashboards controller index view
 module DashboardsHelper
-  def week_range(date: DateTime.now.in_time_zone, start_day: :monday)
+  def week_range(date: Time.zone.now, start_day: :monday)
     week_start = date.beginning_of_week start_day
     week_end = date.end_of_week start_day
-    week_start_string = week_start.strftime('%B %-d')
+    week_start_string = l week_start, format: '%B %-d'
     week_end_string = if week_start.month == week_end.month
-                        week_end.strftime('%-d')
+                        l week_end, format: '%-d'
                       else
-                        week_end.strftime('%B %-d')
+                        l week_end, format: '%B %-d'
                       end
     "#{week_start_string} - #{week_end_string}"
   end
 
   def voicemail_options
-    enum_text = { not_specified: 'No instructions; no ID VM',
-                  no: 'Do not leave a voicemail',
-                  yes: 'Voicemail OK, ID OK' }
-    vm_options = Patient.voicemail_preference.values
-    vm_options.map! { |option| option.to_sym }
-
-    vm_options.map { |option| [enum_text[option], option.to_sym] }
+    enum_text = { not_specified: t('dashboard.helpers.voicemail_options.not_specified'),
+                  no: t('dashboard.helpers.voicemail_options.no'),
+                  yes: t('dashboard.helpers.voicemail_options.yes') }
+    Patient.voicemail_preference
+           .values
+           .map { |option| option.to_sym }
+           .map { |option| [enum_text[option], option.to_sym] }
```

Comment:
Suggest maybe an explainer comment for why we're doing a series of maps? It's not super complicated, but it does take reading a couple times to parse. Might be worth having the intent spelled out to avoid 'fixes' later.

Reply:
good catch! I'll add a 'here's what we're doing' comment.